assignment_repos/Assignment02/test/runtests.jl

```diff
@@ -78,4 +78,4 @@ end
     @test isapprox(bookprice(1, 0, 1), 4.0)
 end
 
-end # "Assignment02"()
\ No newline at end of file
+end # "Assignment02"()
```

Comment:
Can you remove the parens too? Not sure why those are there, but they're unnecessary